src/App.tsx

```diff
@@ -54,6 +53,7 @@ function App() {
     <>
       <RouterProvider router={router} />
       <ConnectWalletModal show={showConnectModal} />
+      <CreateAntModal />
```

Comment:
hmm - thought we were nested this in the `Create` page

Reply:
If we nest it in the create page then the user has to navigate to the create page, no?